addons/mail/static/src/discuss/discuss.js

```diff
@@ -16,8 +17,18 @@ export class Discuss extends Component {
 
     setup() {
         this.messaging = useMessaging();
+        this.state = useState({ pendingThreadName: "", isHoveringThreadName: false });
```

Comment:
not sure what @alexkuhn or @seb-odoo think, but maybe it's time to introduce a new sub component for discuss, the topbar

Reply:
We'll eventually introduce it. In this PR, not a big deal because `Discuss` component is still quite easy to understand.
(_when taking comments into account, JS would still be below 100 LOCs, and template barely above `50` or below)_